extension/k8sleaderelector/extension.go

```diff
@@ -14,19 +14,19 @@ import (
 )
 
 type (
-	StartCallback = func(context.Context)
-	StopCallback  = func()
+	StartCallBack = func(context.Context)
```

Comment:
can these renames be done elsewhere? They are not related to this change.

Reply:
reverted